core/domain/user_services.py

```diff
@@ -1653,12 +1897,15 @@ def get_user_contributions(user_id, strict=False):
             model.id, model.created_exploration_ids,
             model.edited_exploration_ids)
     else:
-        result = None
+        return None
```

Comment:
It may make more sense to do: if model is None: return None, and then the main logic in the main body of the function.